src/Loop/Driver.php

```diff
@@ -40,7 +40,7 @@ public function defer(callable $callback, $data = null);
     /**
      * Delay the execution of a callback.
      *
-     * The delay is a minimum and approximate, accuracy is not guaranteed.
+     * The delay is a minimum and approximate, accuracy is not guaranteed. Order of calls MUST be determined by which timers expire first.
```

Comment:
As we don't have microsecond resolution, just millisecond, the first scheduled one should be executed first.